### PR TITLE
Update kubeedge guide

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: Sherlock113 <sherlockxu@yunify.com>

### CODE REVIEW

File: content/en/docs/installing-on-linux/cluster-operation/add-edge-nodes.md [hunk 1]
```diff
@@ -122,6 +128,37 @@ To make sure edge nodes can successfully talk to your cluster, you must forward
    After an edge node is added, if you cannot see CPU and memory resource usage on the **Edge Nodes** page, make sure [Metrics Server](../../../pluggable-components/metrics-server/) 0.4.1 or later is installed in your cluster.
 
    {{</ notice >}}
+   
+6. After an edge node joins your cluster, some Pods may be scheduled to it while they remains in the `Pending` state on the edge node. Due to the tolerations some DaemonSets (for example, Calico) have, you need to manually patch some Pods so that they will not be schedule to the edge node.
```
Comment (zhu733756) [inline content/en/docs/installing-on-linux/cluster-operation/add-edge-nodes.md:132]: This script was a selectable option. The official should propose optimization  oneday. So we had better add a flag "optional".

Comment (Sherlock113) [reply content/en/docs/installing-on-linux/cluster-operation/add-edge-nodes.md]: Fixed.

**Pull request merged**